powermax/provider/storagegroup_resource.go

```diff
@@ -72,6 +75,14 @@ func (r *StorageGroup) Schema(ctx context.Context, req resource.SchemaRequest, r
 				Required:            true,
 				Description:         "The name of the storage group",
```

Comment:
@doriac11 can you update the description and so related documentation generation?

Reply:
Yes will do!

Reply:
Done